openlcs/reports/serializers.py

```diff
@@ -116,3 +119,17 @@ def get_success_scans(self, obj):
         components = obj.get_synced_components()
         purls = [component.purl for component in components]
         return len(purls)
+
+    def get_complete_scans(self, obj):
+        subscription_id = obj.id
+        task_objs = Task.objects.filter(
+            params__contains=f'subscription_id\": {subscription_id}'
+        )
+        count = 0
+        for task_obj in task_objs:
+            task_status = task_obj.status
+
+            if task_status in ["DUPLICATE", SUCCESS, FAILURE]:
+                count += 1
+
+        return count
```

Comment:
Got it. But i think add a new variable **tasks** will cause extra memory used because tasks can be a very large list if there is many components in this subscription.